x-pack/plugins/spaces/public/lib/constants.ts

```diff
@@ -4,8 +4,26 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
 import chrome from 'ui/chrome';
 
-export const SPACES_FEATURE_DESCRIPTION = `Organize your dashboards and other saved objects into meaningful categories.`;
+let SpacesFeatureDescription;
```

Comment:
let's rename `SpacesFeatureDescription` => `spacesFeatureDescription`. Specifying like this should eliminate your build error too (you'd need to rename the variable usage below as well):

```suggestion
let spacesFeatureDescription: string;
```

Reply:
Fixed, thanks!